tools/permuter_settings.toml

```diff
@@ -11,6 +11,7 @@ CLOSE_DISPS = "void"
 ABS = "int"
 SQ = "int"
 CLAMP = "int"
+ALIGN16 = "int"
```

Comment:
I personally don't think we should have align macros preserved by the permuter, especially since we know they didn't use them consistently everywhere